code/modules/research/designs/misc_designs.dm

```diff
@@ -289,6 +289,16 @@
 ////////////Janitor Designs//////////////
 /////////////////////////////////////////
 
+/datum/design/broom
+	name = "Broom"
+	desc = "Just your everyday standard mop."
+	id = "broom"
+	build_type = PROTOLATHE
+	materials = list(MAT_METAL = 1000, MAT_GLASS = 600)
```

Comment:
Why not plastic and not glass?

Reply:
Because plastic is annoying, bit I did miss that description.